release-notes/1.0/releases.json

```diff
@@ -0,0 +1,1261 @@
+                    "url": "https://download.microsoft.com/download/0/f/1/0f1a7818-dae3-437f-a063-3cc62b840927/dotnet-win-x64.1.0.12.exe", 
+                    "hash": "0eedfca647ef551b19243eaec5417393562f5b1c9bbc920d91db8cc0a192543c59237980f0b2c5d388ffc89c4cb121b2acd75c1e21c2bdf9daafeeb9780b28f4"
+                },
+                {
+                    "name": "dotnet-centos-x64.tar.gz", 
+                    "url": "https://download.microsoft.com/download/0/f/1/0f1a7818-dae3-437f-a063-3cc62b840927/dotnet-centos-x64.1.0.12.tar.gz", 
+                    "hash": "3758c4a1b3e853864ef955ad9cee5f852e484f896d7a05c3994bb6f9b2b487c248577c3fd30c35eed674de649c1c8d0de709e432b0b2a7ccda9b81ed1f1c4f22"
+                },
+                {
+                    "name": "dotnet-debian-x64.tar.gz", 
+                    "url": "https://download.microsoft.com/download/0/f/1/0f1a7818-dae3-437f-a063-3cc62b840927/dotnet-debian-x64.1.0.12.tar.gz", 
+                    "hash": "a69bfa909454b5de50cb177f00528a7f05641f823e1aa30569fc9357d50cc3cc22a2dae6ca806fb8300345765cb4873b9884fd42807d922a4a2476f1b8cc72ea"
+                },
+                {
+                    "name": "dotnet-ubuntu-x64.tar.gz", 
+                    "url": "https://download.microsoft.com/download/0/f/1/0f1a7818-dae3-437f-a063-3cc62b840927/dotnet-ubuntu-x64.1.0.12.tar.gz", 
+                    "hash": "338c1dbd127205d6e10dccf51ca4ac00685a3062aaf2297bac6b78771a0dad37bd7da0d9991e14c09ca90ff6066c00584d90848eeaf816ab6cb5a871956410e9"
+                },
+                {
+                    "name": "dotnet-ubuntu.16.04-x64.tar.gz", 
+                    "url": "https://download.microsoft.com/download/0/f/1/0f1a7818-dae3-437f-a063-3cc62b840927/dotnet-ubuntu.16.04-x64.1.0.12.tar.gz", 
+                    "hash": "e0122f97adfde7a30ce1c251e7ec62700e7e02083ac94b445d5708b924bec99304f280068c345f9352f0ed20ceb83f77fee05553264e2768d669dc63c6872b76"
+                }
+            ]
+        },
+        "sdk":
+        {
+            "version": "1.1.10",
+            "files": 
+            [
+                {
+                    "name": "dotnet-dev-osx-x64.tar.gz", 
+                    "url": "https://download.microsoft.com/download/9/e/6/9e6e1700-f682-4e4c-9e02-583f102cb048/dotnet-dev-osx-x64.1.1.10.tar.gz", 
+                    "hash": "bf3bf5f22c70c944a9459565626ef935cd15465a189ef9dd65d18560f7e71e44e2ac93552999774eb9a6672530a99baa30347bf3ea1cc7f4ac4d583e1f0e60b9"
+                },
+                {
+                    "name": "dotnet-dev-osx-x64.pkg", 
+                    "url": "https://download.microsoft.com/download/9/e/6/9e6e1700-f682-4e4c-9e02-583f102cb048/dotnet-dev-osx-x64.1.1.10.pkg", 
+                    "hash": "990d54a53702f5314ad06aab3fa7b6ad0944f2d8863d5ba99f9e86a660fc542378e0fa2cf9aebbd44bdee71bdd37d287409ded40f98d4a43f77e613a3d84fee3"
+                },
+                {
+                    "name": "9e02-9e02-583f102cb048/dotnet-dev-w.zip", 
```

Comment:
There are move of these incorrect names.

Reply:
search for `dotnet-dev-wizip` and you will find a number of them.  I'm not sure if there are more as well.

Reply:
Sigh, the hazards of hand editing / pattern matching. Fixed up these instances then hit on a good way to view all "name" fields in a contiguous list which made reviewing everything much easier. Found a handful of other patterns that were incorrect and a few other errors in the process. Updated the 2.0 file as penance for being sloppy.